tests/test_extensions.py

```diff
@@ -1,4 +1,5 @@
 from rest_framework import fields, serializers, viewsets, mixins
+from rest_framework_gis.fields import GeometryField, GeometrySerializerMethodField
```

Comment:
I notice that tests are not being invoked yet, possibly because it is a draft WIP.  This would have failed.

I wasnt sure whether to add it as a test dependency, but it looks like https://github.com/openwisp/django-rest-framework-gis/blob/master/setup.py doesnt introduce any additional dependencies, so it is a small overhead to the test bed.

However I am open to doing an optional import, and skip the tests if the import wasnt successful.

Reply:
yes. travis does not do the WIP i believe.

you can add the dep to https://github.com/tfranzel/drf-spectacular/blob/master/requirements/optionals.txt . this is only used for testing. direct imports in tests are fine. lib code must run without the package though (nested imports)